include/task.h

```diff
@@ -23,6 +23,7 @@ namespace detail {
 		collective,     ///< host task with implicit 1d global size = #ranks and fixed split
 		master_node,    ///< zero-dimensional host task
 		horizon,        ///< task horizon
+		fence,
```

Comment:
While some of the comments on the task_types above are meaningless, not having one for `epoch` and `fence` offends my sense of propriety.

Reply:
Commented.